packages/react-components/react-swatch-picker-preview/src/components/ColorSwatch/useColorSwatchState.tsx

```diff
@@ -0,0 +1,19 @@
+// import * as React from 'react';
+import { swatchCSSVars } from './useColorSwatchStyles.styles';
+import type { ColorSwatchProps, ColorSwatchState } from './ColorSwatch.types';
+
+const { swatchColor } = swatchCSSVars;
+
+export const useColorSwatchState_unstable = (state: ColorSwatchState, props: ColorSwatchProps) => {
+  const { color = '#000' } = props;
```

Comment:
```suggestion
  const { color } = props;
```

`color` is a required prop, no?